samples/ReadMe/pacts/Something API Consumer-Something API.json

```diff
@@ -0,0 +1,44 @@
+{
+  "consumer": {
+    "name": "Something API Consumer"
+  },
+  "interactions": [
+    {
+      "description": "A GET request to retrieve the something",
+      "providerStates": [
+        {
+          "name": "There is a something with id 'tester'"
```

Comment:
We should probably use the "provider state with param" option instead of hard-coding the ID in here, since we'd expect that to be best practice from v3 onwards.

Reply:
Could you provide a sample here? This looks similar to the pact-go docs where they used "A user with ID 10 exists".